desktop/src/renderer/src/components/SideNav.tsx

```diff
@@ -29,22 +30,32 @@ export const SideNav = ({ accessToken, idToken }: Props) => {
 	};
 
 	const NavItem = ({ path, label, icon }: NavItemProps) => {
+		const theme = useTheme();
 		return (
 			<ListItem
 				sx={{
 					gap: 1,
 					border: "none",
 					borderRadius: "5px",
-					background: currentPage === path ? "#ececec" : "#f9f9f9",
+					background:
+						currentPage === path ? theme.palette.secondary.light : theme.palette.secondary.main,
 					"&:hover": {
-						background: "#ececec",
+						background: theme.palette.secondary.dark,
 					},
 				}}
 				component="button"
 				onClick={() => handleSelection(path)}
 			>
-				{icon}
-				<Typography sx={{ fontWeight: 600, fontSize: "1em", color: "#6e6e6e" }}>{label}</Typography>
+				<Typography sx={{ color: `${theme.palette.secondary.dark}` }}>{icon}</Typography>
+				<Typography
+					sx={{
+						fontWeight: `${theme.typography.h4.fontWeight}`,
```

Comment:
If you want to use the theme font size and weight its better to add the prop `variant="h4"`